### ISSUE TITLE
"sfdx force:source:deploy -x path/to/package.xml" is not deploying the fields

### ISSUE THREAD

**Issue by hemendrarajawat** [forcedotcom/cli#639]
### Summary

`sfdx force:source:deploy -x path/to/package.xml` is not deploying the fields present in the force-app folder and in the manifest file until specifies directly using the path, i.e., `sfdx force:source:deploy -p path/to/source`. I am encountering this issue only after updating to "sfdx-cli/7.74.1-32db2396ed" version.

### Steps To Reproduce:

1. Retrieve the fields from one sandbox using manifest file: `sfdx force:source:retrieve -x manifest/package.xml -u org1`
2. Deploy the retrieved package to the second sandbox using `sfdx force:source:deploy -x manifest/package.xml -u org2`

### Expected result

The fields should be deployed to the org2 sandbox and the name of the fields should be included in the final "Deployed Source" list. 

### Actual result

Fields were not deployed to the org2 sandbox and the names are missing from the "Deployed Source" list also. 

### Additional information

_Feel free to attach a screenshot_.

**SFDX CLI Version**(to find the version of the CLI engine run sfdx --version):
`sfdx-cli/7.74.1-32db2396ed win32-x64 node-v12.18.3`

**SFDX plugin Version**(to find the version of the CLI plugin run sfdx plugins --core)
```
@oclif/plugin-autocomplete 0.1.5 (core)
@oclif/plugin-commands 1.3.0 (core)
@oclif/plugin-not-found 1.2.4 (core)
@oclif/plugin-plugins 1.9.0 (core)
@oclif/plugin-warn-if-update-available 1.7.0 (core)
@oclif/plugin-which 1.0.3 (core)
@salesforce/analytics 0.22.0
@salesforce/lwc-dev-server 2.5.1
├─ @oclif/plugin-update 1.3.9
└─ @oclif/plugin-help 2.2.3
@salesforce/sfdx-diff 0.0.6
@salesforce/sfdx-trust 3.4.3 (core)
alias 1.1.1 (core)
analytics 1.12.1 (core)
config 1.1.8 (core)
generator 1.1.3 (core)
isvte-sfdx-plugin 1.1.6
salesforcedx 49.10.0 (core)
├─ templates 49.4.4 (core)
├─ @salesforce/sfdx-plugin-lwc-test 0.1.7 (core)
├─ custom-metadata 1.0.10 (core)
├─ apex 0.0.9 (core)
└─ salesforce-alm 49.11.0 (core)
sfdx-cli 7.74.1 (core)
```
**OS and version**:
`Windows 10 Pro 1909`

**Comment by SCWells72** [comment]
I'm also seeing this issue, as are users of Illuminated Cloud working with source format projects against non-scratch orgs. If you try to deploy a single field using the CLI with JSON output, it yields:

```json
{
  "status": 0,
  "result": {
    "deployedSource": []
  }
}
```

If you use the `-w 0` flag, the response does not include an async job ID, so it just appears as if it's not processing the fields at all.

**Comment by SCWells72** [comment]
To expand on my previous comment--and likely be more consistent with the OP--this isn't just for single field deployment. It happens anytime that a field is requested for deployment via the CLI using a `package.xml` vs. deployment via source paths or explicit metadata type/name combinations.

**Comment by shetzel** [comment]
This is essentially a duplicate of #623, #628, #625, #156, #628. This is fixed and it will be in the next patch release.

**Comment by SCWells72** [comment]
Thanks, @shetzel. What's the ETA of the next patch release. As you can hopefully appreciate, this has a significant impact on end users.

**Comment by shetzel** [comment]
ETA for next patch release is Thursday, Oct 1st.  I understand the urgency with several of the fixes queued up.  Believe me, we're doing our best to get them to you without regressions.

**Comment by narensfdc8** [comment]
Still having same issue

**Comment by shetzel** [comment]
This is fixed with CLI version 7.76.1 and salesforcedx plugin version 50.1.1

**Comment by vnehess** [comment]
This is fixed in CLI version 7.76.1 and salesforcedx plugin version 50.1.1, i have a different issue now...

**Comment by shetzel** [comment]
@vnehess - I'm not able to repro this anymore and everyone I've communicated with in the community has told me this is fixed in 7.76.1.  If it's not working for you then I'd ensure you're using salesforcedx plugin v50.1.1.  If you've already done this please open a new issue with the repro steps as I'm not able to repro this with the steps in this issue.

**Comment by vnehess** [comment]
Thanks for your review, my retest encountered a different issue which I assumed was this problem.  I was too quick to assume that this issue was still open.  

It appears that I am not able to deploy any source which has an email folder
https://github.com/forcedotcom/cli/issues/612
